bp/src/SimulatorHelper.m

```diff
@@ -65,11 +65,12 @@ + (NSDictionary *)appLaunchEnvironmentWithBundleID:(NSString *)hostBundleID
     NSString *hostAppExecPath = [SimulatorHelper executablePathforPath:config.appBundlePath];
     NSString *hostAppPath = [hostAppExecPath stringByDeletingLastPathComponent];
     NSString *testSimulatorFrameworkPath = [hostAppPath stringByDeletingLastPathComponent];
-    NSString *libXCTestBundleInject = [[hostAppPath stringByAppendingPathComponent:@"Frameworks"] stringByAppendingPathComponent:@"libXCTestBundleInject.dylib"];
+    NSString *libXCTestBundleInjectPath = [[hostAppPath stringByAppendingPathComponent:@"Frameworks"] stringByAppendingPathComponent:@"libXCTestBundleInject.dylib"];
+    NSString *libXCTestBundleInjectValue = [NSFileManager.defaultManager fileExistsAtPath:libXCTestBundleInjectPath] ? libXCTestBundleInjectPath : @"";
```

Comment:
Can you throw a debug message here saying the dyib didn't exist? Something like 
```suggestion
    NSString *libXCTestBundleInjectValue = [NSFileManager.defaultManager fileExistsAtPath:libXCTestBundleInjectPath] ? libXCTestBundleInjectPath : @"";
    [BPUtils printInfo:DEBUGINFO withString:@"Injecting dylib: %@.", libXCTestBundleInjectValue];
    
```

Reply:
Good call, updated.